src/components/HomeSearchBox/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react'
+import { useRouter } from 'next/router'
+import useTranslation from 'next-translate/useTranslation'
+
+const HomeSearchBox = () => {
+  const { t } = useTranslation('home')
+  const router = useRouter()
+  const [searchValue, setSearchValue] = useState<string>('')
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchValue(event.target.value)
+    console.log(searchValue)
+  }
+
+  const handleSearchClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    router.push('products/${searchValue}')
+    setSearchValue('')
+  }
+
+  const handleKeyPress = (event: React.KeyboardEvent<HTMLInputElement>) => {
+    if (event.key === 'Enter' && searchValue) {
+      handleSearchClick()
+    }
+  }
+
+  return (
+    <div className="home-search-box">
+      <input
+        value={searchValue}
+        type="search"
+        onChange={handleChange}
+        onKeyUp={handleKeyPress}
+        placeholder={t('searchPlaceholder')}
+        name="text"
```

Comment:
Do we really need this?

Reply:
According to [this article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input):  "Consider the name a required attribute (even though it's not). If an input has no name specified, or name is empty, the input's value is not submitted with the form"